language.md

```diff
@@ -16,9 +16,9 @@ Daca vorbesti romana, [da click aici](romanian/nalba.md).
 
 Ifyay ouyay eakspay Igpay Atinlay, [ogay erehay](pig-latin/arshmallowmay.md).
 
-Si vous parlez français, [suivez-moi] (french/feu-de-camp.md)

```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.